src/main/java/xyz/aadev/generitech/common/blocks/power/BlockPowerStorage.java

```diff
@@ -55,10 +55,14 @@ public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, En
         TileEntity tileEntity;
         tileEntity = world.getTileEntity(pos);
         GeneriTech.Logger.info(Long.toString(((TileEntityPowerStorage) tileEntity).getStoredPower()));
+        int[] test = ((TileEntityPowerStorage) tileEntity).getSides();
+        System.out.println(test[0]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS106)